docs/new_exercise_issue_template.md

```diff
@@ -0,0 +1,10 @@
+There is a new exercise, [exercise-name](https://github.com/exercism/problem-specifications/blob/master/exercises/exercise-name/description.md), which data can be found here: https://github.com/exercism/problem-specifications/tree/master/exercises/exercise-name
```

Comment:
IMO it would be nice to have `EXERCISE-NAME` all in caps here and throughout this file. That way you can easily find all of the instances that need to be replaced.

Reply:
Got it, will make the changes